tests/demographic/test_demographic_resources.py

```diff
@@ -190,117 +194,78 @@ def test_get_all(self):
-                                           kf_id=kwargs['kf_id']),
-                                   headers=self._api_headers(),
-                                   data=json.dumps(body))
+        response = self.client.patch(url_for(DEMOGRAPHICS_URL,
+                                             kf_id=kf_id),
+                                     headers=self._api_headers(),
+                                     data=json.dumps(body))
         # Check status code
         self.assertEqual(response.status_code, 200)
-        # Check field values got updated
-        response = json.loads(response.data.decode('utf-8'))
-        demographic = response['results']
-        self.assertEqual(kwargs['kf_id'], demographic['kf_id'])
-        # Fields that should be None since they were not in put request body
-        self.assertIs(None, demographic['external_id'])
-        self.assertIs(None, demographic['ethnicity'])
-        # Fields that should be updated w values
-        self.assertEqual(body['race'], demographic['race'])
-        self.assertEqual(body['gender'], demographic['gender'])
-
-    def test_put_not_found(self):
-        """
-        Test update non-existent demographic
-        """
-        # Send put request
-        kf_id = 'non-existent'
-        body = {
-            'race': 'black or african',
-            'gender': 'male',
-            'participant_id': id_service.kf_id_generator('PT')()
-        }
-        response = self.client.put(url_for(DEMOGRAPHICS_URL,
-                                           kf_id=kf_id),
-                                   headers=self._api_headers(),
-                                   data=json.dumps(body))
-        # Check status code
-        self.assertEqual(response.status_code, 404)
-        # Check response body
-        response = json.loads(response.data.decode("utf-8"))
-        # Check error message
-        message = 'could not find demographic'
-        self.assertIn(message, response['_status']['message'])
-        # Check database
-        c = Demographic.query.filter_by(kf_id=kf_id).count()
-        self.assertEqual(c, 0)
 
-    def test_put_bad_input(self):
+        # Message
+        resp = json.loads(response.data.decode("utf-8"))
+        self._test_response_content(resp, 200)
+        self.assertIn('demographic', resp['_status']['message'])
+        self.assertIn('updated', resp['_status']['message'])
+
+        # Content - check only patched fields are updated
+        demographic = resp['results']
+        dm = Demographic.query.get(kf_id)
+        for k, v in body.items():
+            self.assertEqual(v, getattr(dm, k))
+
+        # Content - Check remaining fields are unchanged
+        unchanged_keys = (set(demographic.keys()) -
+                          set(body.keys()))
+        for k in unchanged_keys:
+            val = getattr(dm, k)
+            if isinstance(val, datetime):
+                d = val.replace(tzinfo=tz.tzutc())
```

Comment:
It adds a timezone to `val` since `val` doesn't have one and then returns a new datetime obj. 

https://stackoverflow.com/questions/7065164/how-to-make-an-unaware-datetime-timezone-aware-in-python